pkg/metrics/metrics.scss

```diff
@@ -284,23 +289,41 @@
         grid-row: 0;
         grid-column: 1 / -1;
         background: #eee;
+        margin-left: var(--pf-global--spacer--md);
     }
 
     &-events {
         grid-column: events;
         grid-row: calc(var(--metrics-minutes) - var(--metrics-minute));
-        display: grid;
-        grid-template-columns: [info] 1fr [time] 1fr;
-        align-items: baseline;
 
-        dt {
-            grid-column: time;
-            font-size: var(--pf-global--FontSize--sm);
-            grid-row: 1;
+        .description {
+            display: grid;
+            grid-template-columns: [info] 1fr [time] 1fr;
+            align-items: baseline;
+            justify-items: start;
+
+            .time {
+                grid-column: time;
+                font-size: var(--pf-global--FontSize--sm);
+                grid-row: 1;
+                display: flex;
+                justify-content: space-between;
+                width: 100%;
+            }
+
+            .type {
+                grid-column: info;
+            }
         }
 
-        dd {
-            grid-column: info;
+        .cockpit-log-panel {
```

Comment:
What's the purpose of the content of this rule? I removed it and I did not notice any change. If it's needed can we add some comment?

Reply:
To have "Loading..." and "No logs found" in the middle